bottlecap/src/metrics/aggregator.rs

```diff
@@ -291,10 +220,11 @@ impl<const CONTEXTS: usize> Aggregator<CONTEXTS> {
     }
 
     fn build_sketch(&self, now: i64, entry: &Entry) -> Option<Sketch> {
-        if entry.kind != metric::Type::Distribution {
+        if let MetricValue::Distribution(_) = entry.metric_value {
+        } else {
             return None;
-        }
-        let sketch = entry.metric_value.get_sketch();
+        };
+        let sketch = entry.metric_value.get_sketch()?;
```

Comment:
I don't understand this change, you're not using the value, why change the logic here?

Reply:
I think it's due to how logic was modified to instead of checking the kind, now this is "the way" of checking which type of metric it is. Is there a more idiomatic way of doing this?